R/lmabc.R

```diff
@@ -93,7 +93,7 @@
 lmabc = function(formula, data, ..., props = NULL){
 
 	# Fill in the data argument with stats::model.frame
-	data <- model_frame(formula = formula, data = data)
```

Comment:
Will this run into problems if the "..." argument in lmabc has more terms that are passed in?